### DISCUSSION TITLE
How to make sure a moved package not longer get new tags

### DISCUSSION THREAD

**Discussion by alexander-schranz** [composer/composer#12200] [Q&A / Support]
We renamed a package some time ago to be more consistent:

 - https://packagist.org/packages/sulu/sulu-form-bundle

Which is now:

 - https://packagist.org/packages/sulu/form-bundle
 
The abandoned name (https://packagist.org/packages/sulu/sulu-form-bundle) still gets the new tags, as Github points it still to the same repository. Is there a setting to not longer fetch new tags for abandoned name package names to force update who want the new verson to update there composer.json also? I'm planning to move more packages to a new namespace currently thats why I mostly interested how to achieve this a cleaner forced migration to the new name.

**Comment by stof** [comment]
@Seldaek shouldn't abandoned packages be excluded from the auto-update of Packagist ?

**Comment by Seldaek** [comment]
I don't see why? People get a notice on install.. If they ignore it then what does it achieve to stop them from receiving new versions? Most likely they'll ignore that or not realize it and end up on outdated/buggy/non-secure software. I think it's perfectly fine as it is.
